js/modifyContributionForm.js

```diff
@@ -0,0 +1,83 @@
+CRM.$(function ($) {
+
+  (function() {
+    setTotalAmount();
+    hideStatusField();
+    setAmountCurencySymbol();
+    toggleRecordPaymentBlock();
+    placePaymentFieldsTogether();
+  })();
+
+  function hideStatusField() {
```

Comment:
can you please rearrange the functions below by the order in which they are called above?